default/scripting/species/common/focus.macros

```diff
@@ -57,6 +49,12 @@ HAS_ADVANCED_FOCI
             location = OwnerHasTech name = "SHP_INTSTEL_LOG"
             graphic = "icons/focus/supply.png"
 
+        Focus
+            name = "FOCUS_STOCKPILE"
+            description = "FOCUS_STOCKPILE_DESC"
+            location = OwnerHasTech name = "PRO_GENERIC_SUPPLIES"
```

Comment:
Need to tweak the descriptions in a few places to make it clear this tech is required. Maybe also make it a prereq for Void Prediction...?